### DISCUSSION TITLE
Need to send same header with multiple values in same template

### DISCUSSION THREAD

**Discussion by Vishnuraghav89** [projectdiscovery/nuclei-templates#1378] [Q&A] [answered]
Hi Team,

Greetings of the day!!

I was preparing a template and stuck at point that I need to send same header with multiple values in same template with different different request.
Is it possible??

**Comment by ehsandeep** [comment]
Hi @Vishnuraghav89, 

Yes, you can do that, do you see issues while doing it?

**Reply by Vishnuraghav89** [reply]
Could you please give provide an example template for below scenerio.

Header: value1
Header: value2

{{Baseurl}}

I want that first request need to sent with value 1 and then second request need to sent with value2.

Both request need to sent separately with same header but with different different value in same template only.

**Reply by ehsandeep** [reply, marked as answer]
Here is one example:- https://github.com/projectdiscovery/nuclei-templates/blob/3ef39c173e4cc1937158c6ad9ab30d2f2e7366d1/misconfiguration/put-method-enabled.yaml and there are many in the template project.
